search/search_indexes.py

```diff
@@ -38,6 +38,19 @@ class LearningResourceIndex(indexes.SearchIndex, indexes.Indexable):
     # page because that page is always for a single repository.
     repository = indexes.CharField(faceted=True)
 
+    nr_views = indexes.IntegerField(
+        model_attr="xa_nr_views",
+        faceted=True,
```

Comment:
I actually added it to support future faceting, but I guess they are useless now